### ISSUE TITLE
sdwan_traffic_data_policy_definition | "Loss Correction" = "Packet Duplication" not working

### ISSUE THREAD

**Issue by rrahimm** [CiscoDevNet/terraform-provider-sdwan#255]
We are seeing issues when using Loss Correction" of type "Packet Duplication".

Looks the provider should support one more [type](https://registry.terraform.io/providers/CiscoDevNet/sdwan/latest/docs/resources/traffic_data_policy_definition#type) called "lossProtectPktDup" 

Look at the API samples

When "Loss Correction" is "Packet Duplication"

```
        {
          "type": "lossProtect",
          "parameter": "packetDuplication"
        },
        {
          "type": "lossProtectPktDup",
          "parameter": "packetDuplication"
        }
```        
        
        
When "Loss Correction" is "FEC Always"        
```
    	{
          "type": "lossProtect",
          "parameter": "fecAlways"
        },
        {
          "type": "lossProtectFec",
          "parameter": "fecAlways"
        }
```

When "Loss Correction" is "FEC Adaptive"        
```
        {
          "type": "lossProtect",
          "parameter": "fecAdaptive"
        },
        {
          "type": "lossProtectFec",
          "parameter": "fecAdaptive",
          "value": "1"
        }
```

Terraform error below, as we have currently configured Terraform module with what the provider supported.

```
module.sdwan.sdwan_traffic_data_policy_definition.traffic_data_policy_definition["DP-VPN10-01"]: Creating...
╷
│ Error: Client Error
│ 
│   with module.sdwan.sdwan_traffic_data_policy_definition.traffic_data_policy_definition["DP-VPN10-01"],
│   on .terraform/modules/sdwan/sdwan_centralized_policies.tf line 306, in resource "sdwan_traffic_data_policy_definition" "traffic_data_policy_definition":
│  306: resource "sdwan_traffic_data_policy_definition" "traffic_data_policy_definition" {
│ 
│ Failed to configure object (POST), got error: HTTP Request failed: StatusCode 400, {"error":{"type":"error","message":"Invalid policy","details":"SeqID 1: Invalid
│ parameter string packetDuplication for loss protect forward error correction","code":"POLICY0004"}}
```

**Comment by rrahimm** [comment]
Hello @danischm  and @seconroy 

Can you take a look at this? 

Reported as an issue in https://wwwin-github.cisco.com/netascode/nac-sdwan/issues/176

**Comment by rrahimm** [comment]
Hello @seconroy 

On what version will this be released ?

**Comment by seconroy** [comment]
Will be fixed in 0.3.10.

**Comment by danischm** [comment]
Fixed in v0.3.10.
